demo/index.html

```diff
@@ -19,9 +19,9 @@
     <div class="wsuwrap">
         <div class="wsuwordmark">
             <h1>
-                <a href="https://wayne.edu/">
+                <a href="https://wayne.edu/" aria-labelledby="wsuheader_title">
                     <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 605 65" preserveAspectRatio="xMaxYMin meet">
-                        <title>Wayne State University</title>
+                        <title id="wsuheader_title">Wayne State University</title>
```

Comment:
@breakdancingcat Can we use dashes for `id` and `class` names to match our overall style?